app/services/site_configs/upsert.rb

```diff
@@ -0,0 +1,114 @@
+module SiteConfigs
+  module Upsert
+    include AuthenticationHelper
```

Comment:
it's a bit odd to include a view helper in a service, I wonder if there's a separate way to do this